pycyphal/transport/can/media/pythoncan/_pythoncan.py

```diff
@@ -188,7 +187,12 @@ def __init__(
         self._closed = False
         self._maybe_thread: typing.Optional[threading.Thread] = None
         self._rx_handler: typing.Optional[Media.ReceivedFramesHandler] = None
-        self._background_executor = concurrent.futures.ThreadPoolExecutor(max_workers=1)
+        # This is for communication with a thread that handles the call to _bus.send
+        self._tx_queue: queue.Queue[
+            typing.Optional[typing.Tuple[can.Message, float, asyncio.Future[None], asyncio.AbstractEventLoop]]
```

Comment:
At this point I think we should have a dedicated dataclass for this.

```python
@dataclasses.dataclass(frozen=True)
class _TxItem:
    msg: can.Message
    timeout: float
    future: asyncio.Future[None]
    loop: asyncio.AbstractEventLoop
```

Then define the queue as `queue.Queue[_TxItem | None]`